src/app/controllers/views/ControllerList/ControllerListTable/ControllerListTable.tsx

```diff
@@ -99,7 +97,7 @@ const generateRows = ({
             <span className="u-truncate">
               {controller.node_type === NodeType.REGION_CONTROLLER ||
               controller.node_type === NodeType.REGION_AND_RACK_CONTROLLER ? (
-                vaultEnabled ? (
+                vaultEnabled === true ? (
```

Comment:
What's the reason for this change? `vaultEnabled` is of boolean type anyway, TS will ensure this can be either true or false.

Reply:
Ah that was left in there from some testing and bug fixing. Will change